package.json

```
@@ -127,7 +127,8 @@
     "libnpmconfig": "1.2.1",
     "node-fetch": "2.7.0",
     "npm-registry-fetch": "14.0.5",
-    "semver": "7.6.3"
+    "semver": "7.6.3",
+    "yaml": "^2.5.0"
```

Should version be fixed?

```suggestion
    "yaml": "2.5.0"
```

---

its just a matter of taste but I prefer fixed versions, yeah :P